app/views/capture_email.scala.html

```diff
@@ -0,0 +1,51 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.views.html.helpers.{form, errorSummary, input}
+@import config.AppConfig
+@import forms.EmailForm._
+
+@(emailForm: Form[String])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(Messages("captureEmail.title")) {
+
+  @errorSummary(messages("common.error.heading"), emailForm, forceDivToAppear = false)
+
+  <h1 id="main-heading" class="form-title heading-xlarge">@messages("captureEmail.title")</h1>
```

Comment:
I don't think we need the id on here. 
What does the form-title css add? Is it used in form validation or something?
Can `heading-xlarge` be set to `heading-large` to match the prototype.